__test__/schema/fixtures/credential-test:IdDocument-v1.schema.json

```diff
@@ -5,8 +5,7 @@
   "type": "object",
   "allOf": [
     {
-      "$ref": "http://identity.com/schemas/credential-cvc:IdDocument-v2"
+      "$ref": "http://identity.com/schemas/credential-cvc:IdDocument-v3"
     }
-  ],
-  "additionalProperties": false
```

Comment:
I removed this temporarily. AJV isn't playing nice since the merging in [IDCOM-1592](https://civicteam.atlassian.net/jira/software/c/projects/IDCOM/boards/123?selectedIssue=IDCOM-1592)

Will fix in a seperate PR